app/models/event.py

```diff
@@ -125,6 +125,13 @@ class Event(SoftDeletionModel):
         'EventSubTopic', backref='event', foreign_keys=[event_sub_topic_id])
     events_orga = db.relationship(
         'EventOrgaModel', backref='event', foreign_keys=[events_orga_id])
+    owner = db.relationship('User',
```

Comment:
this will return an array/list with a single user inside it, check docs of alchemy to restrict it to a single result.

Reply:
@CosmicCoder96 I've used - `owner = db.relationship('User', backref='owner_events')` now